bin/start_do.sh

```diff
@@ -160,7 +160,30 @@ else
   echo "Create a password for the grafana admin user"
   pulumi config set prometheus:adminpass -C ${script_dir}/../pulumi/python/config
 fi
+# TODO: Figure out better way to handle hostname / ip address for exposing our IC #82
+#
+# This version of the code forces you to add a hostname which is used to generate the cert when the application is
+# deployed, and will output the IP address and the hostname that will need to be set in order to use the self-signed
+# cert and to access the application.
+#
+
+echo " "
+echo "NOTICE! Currently we do not automatically pull the hostname of the K8 LoadBalancer with this deployment; instead"
```

Comment:
Is there a URL with documentation on this in which you can point people?

Reply:
Fixed